### ISSUE TITLE
Copy Pods Resource Script using Xcode 7+ environment variable

### ISSUE THREAD

**Issue by maschall** [CocoaPods/CocoaPods#5577]
* [x] I've read and understood the [*CONTRIBUTING guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?
Converted from 0.37.1 to 1.0.1
pod lib lint --sources=PrivateRepo,master --allow-warnings --use-library

## What did you expect to happen?
Lint would succeed

## What happened instead?
Xcode error:

```
ibtool --reference-external-strings-file --errors --warnings --notices --minimum-deployment-target  --output-format human-readable-text --compile /Users/mark/Library/Developer/Xcode/DerivedData/App-eayhfhykaspqgeepsdjnpyoabpja/Build/Products/Release-iphonesimulator/App.app/OSAlertActionTableViewCell.nib /var/folders/t5/6gdjdxs525x1970h1t2cjkr00000gn/T/CocoaPods/Lint/Pods/OnStarWidgets/OnStarWidgets/OSAlertActionTableViewCell.xib --sdk /Applications/Xcode_6.4.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator8.4.sdk --target-device ipad --target-device iphone
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>com.apple.ibtool.errors</key>
	<array>
		<dict>
			<key>description</key>
			<string>Unrecognized commandline options: "human-readable-text" and "/var/folders/t5/6gdjdxs525x1970h1t2cjkr00000gn/T/CocoaPods/Lint/Pods/OnStarWidgets/OnStarWidgets/OSAlertActionTableViewCell.xib". Either unrecognized options were passed, or multiple input documents were passed when only one was expected.</string>
		</dict>
	</array>
</dict>
</plist>
```

## CocoaPods Environment

```
   CocoaPods : 1.0.1
        Ruby : ruby 2.2.1p85 (2015-02-26 revision 49769) [x86_64-darwin14]
    RubyGems : 2.4.6
        Host : Mac OS X 10.11.5 (15F34)
       Xcode : 6.4 (6E35b)
         Git : git version 2.3.2 (Apple Git-55)
Ruby lib dir : /Users/mark/.rvm/rubies/ruby-2.2.1/lib
```

If I run `export` in a build phase in Xcode 6.4, the variable `DEPLOYMENT_TARGET_SETTING_NAME` is no where to be found.  I believe this is a Xcode 7+ variable.

I see issue https://github.com/CocoaPods/CocoaPods/issues/5397 and agree that we should keep the `--minimum-deployment-target` argument, but we should either say Xcode 7+ support only or look at checking if `DEPLOYMENT_TARGET_SETTING_NAME` is defined and falling back if not.

Looking for feedback before implementing a solution.

**Comment by neonichu** [comment]
That's unfortunate. I think considering that we usually support the last two versions of Xcode in CocoaPods, it would make sense fixing this — however, we're relatively close to Xcode 8 being released and therefore pushing 6.4 off the list of versions we'd normally support.

**Comment by benasher44** [comment]
Since upgrading CocoaPods to use fourflusher 2.0, it no longer supports Xcode 6 officially. Going to close this.
